### PR TITLE
Update tests to use JavatestUtil

### PR DESCRIPTION
**Pull request by annaibm**
- This code updates the custom tests to use JavatestUtil instead of JavaTestRunner.

- Removes JavaTestRunner file

resolves: https://github.com/adoptium/aqa-tests/issues/4864

### CODE REVIEW

**Review by LongyuZhang — APPROVED** [review]
Related to reverted  PR https://github.com/adoptium/aqa-tests/pull/4893. 
It was reverted because the release test failed after that PR merged with error information related to the packages modified in the PR, so reverted to unblock the releases process, but it was a coincidence, the failure was actually not related to this PR.

**Review by llxia — APPROVED** [review]

**Pull request merged**
